cadasta/gui/tools/wizard/step_project_creation03.py

```diff
@@ -196,6 +199,99 @@ def upload_locations(self):
                 tr('Finish with %d failed' % failed)
             )
 
+    def _url_post_parties(self):
+        """Get url to create a new party.
+
+        :returns: api url
```

Comment:
`returns` -> `return`.
I think it is better with a small sentence or at least something like `The API URL or None if project_upload_result is not set.